changelog/v1.9.0-beta14/nested-claims-docs.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/2744
+    description: Add documentation for nested claims (rather than just top-level claims) in JWT RBAC.
```

```suggestion
    description: Adds documentation for nested claim support in RBAC policies for JSON Web Tokens (JWT).
```

Do you typically add links from the changelog to the topic as well? Such as `... [JSON Web Tokens (JWT) RBAC policies]({{% versioned_link_path fromRoot="/docs/content/guides/security/auth/jwt/access_control/" %}})` , or nah?

---

I don't typically, but that doesn't mean we can't do that going forward.  The bigger question would be: what version of the topic should we link to, and what happens when topics move or get removed?

We could just use whichever version they're currently viewing with, so that's fine. But what happens if a topic is removed or moved to a different url? We would have to either make sure it redirects to correct location, or go back and update links in changelog files.  I don't have a strong opinion here, I feel like a link could be very helpful in figuring out how to use newly released features.  If this is something the tech. writing team would like to see happen moving forward, we should add some information about it to https://github.com/solo-io/go-utils/tree/master/changelogutils#changelog. Perhaps we could add an (optional) `additionalLinks` field to the changelog?